dynamite/plotter.py

```diff
@@ -2092,7 +2093,7 @@ def orbit_distribution(self,
             weight_solver = model.get_weights(orblib)
             weights, _, _, _ = weight_solver.solve(orblib)
         mod_orb_dists = orblib.projection_tensor.dot(weights)
-        mod_orbclass_fracs = np.sum(mod_orb_dists, (1,2))
+        mod_orbclass_fracs = np.sum(mod_orb_dists, (1,2))/np.sum(np.sum(mod_orb_dists, (1,2)))
```

Comment:
Maybe split into two lines for readability? i.e.

```
mod_orbclass_fracs = np.sum(mod_orb_dists, (1,2))
mod_orbclass_fracs = mod_orbclass_fracs/np.sum(mod_orbclass_fracs)
```

Reply:
Done!